lore/end_users/use/README.md

```diff
@@ -73,3 +73,9 @@ what is known in git as a merge conflict.
 
 For information on how to resolve merge conflicts, refer to
 [Resolving a merge conflict on the command line](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/).
+
+**NOTE**: Repositories which require multiple continuous integration checks
+(ex: a repository which has both BonnyCI and Travis CI) will encounter merge
+failures from BonnyCI if the other continuous integration service has checks
+that have not been met. For this reason, the use of other CI services along
+with BonnyCI is discouraged.
```

Comment:
So.... this is not quite true.

One can use multiple CI services alongside BonnyCI. That itself is not ht problem. What _is_ a problem is if in Branch Protection settings, the repository requires successful checks from CI systems other than BonnyCI. That is what will cause the merge failures.

Adding a CI system and requiring it be passing for merges are two separate steps.